tools/Status/Status.py

```diff
@@ -206,32 +215,83 @@ def _format(field: str, value: Any, state_styles: dict) -> str:
             return "-"
         else:
             return humanize.naturaldate(value) + " " + value.strftime("%X")
+    elif field == "SegmentId":
+        if pd.isnull(value):
+            return "-"
+        else:
+            return f"Segment_{int(value):04}"
```

Comment:
I know this needs to be done better, but I couldn't figure out how to get the number of digits from the Directory class.

Reply:
I'd say move the `pd.isnull` down to the catch-all `else` clause. The segment number can just render as an integer, no need to prefix `Segment_`.